src/main/java/io/snyk/agent/logic/FilterUpdate.java

```diff
@@ -79,14 +110,18 @@ private boolean fetchUpdatedAnything() throws IOException {
             lines = reader.lines().collect(Collectors.toList());
         }
 
-        config.filters.set(FilterList.loadFiltersFrom(log, lines));
-        log.info("filters updated, new count: " + config.filters.get().filters.size());
+        final Instant created = Instant.ofEpochMilli(conn.getHeaderFieldDate("Last-Modified", 0));
```

Comment:
just making sure I get this right:
if Homebase returns a snapshot without a `Last-Modified` header then the agent will load it, but set its timestamp to the earliest time possible which will probably get replaced on the next chance?

Reply:
That's the plan. Everything would work much better if homebase didn't do this, though, as we do actually do a load of extra work when the snapshot changes.